src/data/matrix/basic.lean

```diff
@@ -303,58 +303,62 @@ variable [fintype m]
 def dot_product [has_mul α] [add_comm_monoid α] (v w : m → α) : α :=
 ∑ i, v i * w i
 
+/- the precedence of ` ⬝ᵥ ` for `matrix.dot_product` is set as to come
+   immediately after ` • ` for `has_scalar.smul` -/
+localized "infix  ` ⬝ᵥ `:72 := matrix.dot_product" in matrix
```

Comment:
The docstring at the top of the file should mention this new notation.

Reply:
It would be great if you could collect together all the notations in this file when you add it too!

Reply:
Done.